zips/draft-str4d-account-metadata-keys.md

```diff
@@ -0,0 +1,198 @@
+
+    ZIP: Unassigned
+    Title: Account Metadata Keys
+    Owners: Jack Grigg <jack@electriccoin.co>
+            Daira-Emma Hopwood <daira@electriccoin.co>
+            Kris Nuttycombe <kris@electriccoin.co>
+    Status: Draft
+    Category: Standards / Wallet
+    Created: 2025-02-18
+    License: MIT
+
+
+# Terminology
+
+The key words "MUST NOT", "SHOULD", and "MAY" in this
+document are to be interpreted as described in BCP 14 [^BCP14] when, and
+only when, they appear in all capitals.
+
+
+# Abstract
+
+This ZIP specifies the key tree for Account Metadata Keys. These are derived
+from the same seed as a ZIP 32 [^zip-0032] account, and can be used by wallets
+to derive encryption keys for local / off-chain metadata.
+
+
+# Motivation
+
+A wallet's main data source is the Zcash chain: from this it can detect notes
+received by an account, determine whether those notes are spent, build witnesses
+for spending, recover on-chain memo data, and so on. However, wallets also
+generate significant quantities of off-chain metadata as they are used, such as:
+
+- Local notes about transactions in the wallet.
+- Mappings from Zcash addresses to user-meaningful recipient names.
+- The exchange rate from ZEC to another currency that was used to determine how
+  much ZEC to send in a payment.
+
+This metadata is valuable to users, and highly desirable to ensure is backed up.
+If the user's device is wiped or lost and the user recovers their wallet from a
+backed-up mnemonic phrase, they will lose all of this metadata if it is not
+stored somewhere.
+
+For other kinds of phone data, it is expected by users that their phone's normal
+backup storage will have saved (most of) their data, such that access to e.g.
+their Apple or Google account will be sufficient for data recovery. However,
+metadata like mappings from Zcash addresses to recipient names can be
+particularly sensitive, meaning that users may not want it to be backed up
```

Comment:
```suggestion
the associated Apple or Google account will be sufficient for data recovery.
However, metadata-like mappings between Zcash addresses and recipient names can
be particularly sensitive, meaning that users may not want these to be backed up
```

Reply:
"metadata like" was intentional; the mappings are not metadata-like, they _are_ metadata.